apps/insights/src/components/PublisherTag/index.module.scss

```diff
@@ -28,7 +28,7 @@
   .name {
     color: theme.color("heading");
 
-    @include theme.text("base", "medium");
+    @include theme.text("lg", "medium");
```

Comment:
I think this could make the search look weird since publishers & feeds will show up differently, take a look and make sure it looks ok